### DISCUSSION TITLE
Looks like `actions-rs` is unmaintained

### DISCUSSION THREAD

**Discussion by conways-glider** [tokio-rs/axum#1661] [General]
It looks like `actions-rs` is unmaintained (https://github.com/actions-rs/toolchain/issues/216).

It may be worth it to investigate migrating off of it and into more hand-rolled Rust Actions.

**Comment by davidpdrsn** [comment]
Yeah would be good to move away from that. Do you have any recommendations for what we should use instead?

**Comment by FrankReh** [comment]
This appears to be the consensus, looking at the end of that issue.

```
- uses: actions-rs/toolchain@v1
+ uses: dtolnay/rust-toolchain@stable
 ```

**Reply by davidpdrsn** [reply]
Thanks! Fixed in https://github.com/tokio-rs/axum/pull/1687
